assets/js/blocks/mini-cart/utils/data.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import { _n, sprintf } from '@wordpress/i18n';
+import {
+	getCurrencyFromPriceResponse,
+	formatPrice,
+} from '@woocommerce/price-format';
+import { CartResponse } from '@woocommerce/types';
+
+export const updateTotals = ( totals: [ string, number ] | void ) => {
+	if ( ! totals ) {
+		return;
+	}
+	const [ amount, quantity ] = totals;
+	const miniCartButtons = document.querySelectorAll(
+		'.wc-block-mini-cart__button'
```

Comment:
> What do you think about using a more specific selector?

What would be the benefit of using a more specific selector? We want to select all Mini-Cart buttons and `.wc-block-mini-cart__button` will give us all of them.

> Also, is it a particular reason for using `querySelectorAll`?

You mean why using `querySelectorAll` instead of `querySelector`? The reason is that even though we only allow inserting one Mini-Cart button, there is the chance that more than one might be displayed at the same time. Imagine somebody adding the Mini-Cart block to the header of their site and then creating a page with the Mini-Cart block. When viewing that page, there will be two Mini-Cart buttons on the same page. That's why I usually try to make everything work assuming there might be several Mini-Cart blocks on the same, even if we don't officially support that.

Reply:
Yeah! Good point! 👍